moped-etl/arcgis/components_to_agol.py

```diff
@@ -214,6 +260,11 @@ def main(args):
             "Please provide either the -d flag with ISO date string with TZ offset or the -f flag and not both."
         )
 
+    if not args.date and not args.full:
```

Comment:
🙏

Reply:
Thanks for this observation. This change was born out of me forgetting to pick one or the other.